conans/client/tools/version.py

```diff
@@ -42,9 +46,14 @@ def build(self):
     def __eq__(self, other):
         if not isinstance(other, Version):
             other = Version(other)
-        return self._semver.compare(other._semver) == 0
+        return (self._semver.compare(other._semver) or self._compare_micro(other)) == 0
```

Comment:
So the key is that node-semver already had the ``_compare_micro()`` in 0.6.1?

Reply:
No, it has been included in 0.8.0. This is "port" for 0.6.1